UI/installer/mp-installer.nsi

```diff
@@ -421,7 +461,7 @@ SectionEnd
 VIProductVersion "${APPVERSION}.0"
 VIAddVersionKey /LANG=${LANG_ENGLISH} "ProductName" "OBS Studio"
 VIAddVersionKey /LANG=${LANG_ENGLISH} "CompanyName" "obsproject.com"
-VIAddVersionKey /LANG=${LANG_ENGLISH} "LegalCopyright" "(c) 2012-2016"
+VIAddVersionKey /LANG=${LANG_ENGLISH} "LegalCopyright" "(c) 2012-2019"
```

Comment:
Might as well update this too. :P

Reply:
fixed.

Reply:
Why not 2012-2020?

Reply:
You're looking at a comment from an old diff when it still said 2019. The current diff says 2020.

Reply:
In fairness, normally GitHub would hide a conversation from an old diff, but I had clicked on a link from an email...  In the GitHub UI, I thought you had provided a suggested edit with comment instead of commenting on an existing, older diff.  This is why we don't code review right after waking up.